langs/apl/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:bookworm-slim AS builder
```

Comment:
Final comment, sorry. I did try Alpine 3.20, but that's letting me down.

```bash
------
 > [builder 3/3] RUN curl -o dyalog.deb https://www.dyalog.com/uploads/php/download.dyalog.com/download.php?file=18.2/linux_64_18.2.45405_unicode.x86_64.deb  && dpkg -i dyalog.deb:
0.535   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
0.535                                  Dload  Upload   Total   Spent    Left  Speed
100  181M  100  181M    0     0  8592k      0  0:00:21  0:00:21 --:--:-- 9354k
22.30 dpkg: error processing archive dyalog.deb (--install):
22.30  package architecture (amd64) does not match system (musl-linux-amd64)
22.31 Errors were encountered while processing:
22.31  dyalog.deb
------
```
However, I think I found something as we speak.

Reply:
> However, I think I found something as we speak.

Newsflash, I did not.